src/actor.js

```diff
@@ -485,6 +487,68 @@ export const callTask = async (taskId, input, options = {}) => {
     return Object.assign({}, updatedRun, { output });
 };
 
+
+/**
+ * Metamorphs current actor run into a run of another actor with given input.
+ * After this function is called the actor container gets stopped and restarted as a run of given actor.
+ *
+ * @param {String} actorId
```

Comment:
For consistency, please call this `targetActorId`

Reply:
✅